From 99a4da1e9a2be851fd6b6fb603c18a871f5d1fe8 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:15:59 -0400
Subject: [fs] cifs: update cifs.ko version

Message-id: <1406549822-10376-7-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86747
O-Subject: [RHEL7.1 PATCH 06/69] [bz1082049][CIFS] update cifs.ko version
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Steve French <smfrench@gmail.com>

To 2.02

Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit ffe67b58595eda992f369de7ac4d0a9b165074fd)

Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/cifsfs.h b/fs/cifs/cifsfs.h
index 59aa8db..729d31b 100644
--- a/fs/cifs/cifsfs.h
+++ b/fs/cifs/cifsfs.h
@@ -132,5 +132,5 @@ extern long cifs_ioctl(struct file *filep, unsigned int cmd, unsigned long arg);
 extern const struct export_operations cifs_export_ops;
 #endif /* CONFIG_CIFS_NFSD_EXPORT */
 
-#define CIFS_VERSION   "2.01"
+#define CIFS_VERSION   "2.02"
 #endif    /* _CIFSFS_H */
-- 
1.7.1